library/src/jmh/java/bio/terra/pfb/LibraryBenchmarks.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.pfb;
+
+import java.io.IOException;
+import org.openjdk.jmh.annotations.Benchmark;
+
+/**
+ * JMH benchmarks for the Library class.
+ *
+ * <p>Benchmarks do not assert correctness; that's what unit tests are for. These exist only to
```

Comment:
Great commentary here, and I generally agree -- but might it be worth asserting for _some_ kind of success in the event the work _fails quickly_ and falsely looks performant but is really just broken?

Reply:
I added assertions that the results are not empty. I do want to stay away from asserting on correctness - but hopefully this is enough of a safeguard. 

On a related note, these assertions may eliminate a different kind of false positive: if the JVM notices the result of the method call is unused, it could optimize around that during compilation, making the benchmark invalid: https://asgaard.ece.ualberta.ca/papers/Journal/TSE_2019_Costa_Whats_Wrong_With_My_Benchmark_Results_Studying_Bad_Practices_in_JMH_Benchmarks.pdf